Codeinwp/neve
# [Fix] Dynamic label not updating when a new product was added to cart

### Summary
The selector for the cart fragments was wrong. I think we did a refactor at some point and we didn't take this into consideration.

### Will affect visual aspect of the product
NO


### Test instructions
- Add cart Icon component in the header.
- Add any combinations of dynamic labels for it https://prnt.sc/10zvjf5
- Add a product to the cart and see:
1. If the total amount is updating
2. If the label keeps its format ( Eg., If my label is "Hello {cart_total}", after adding a product to the cart the label should show "Hello <new_cart_total>", not just the cart total or the hello part.


Closes Codeinwp/neve-pro-addon#1252.
